deploy/images/bicep/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.21.3
+
+# Argument for target architecture
+ARG TARGETARCH
+
+WORKDIR /
+
+COPY ./linux_${TARGETARCH:-amd64}/release/bicep/bicepconfig.json bicepconfig.json
+
+RUN apk --no-cache add curl
+RUN curl -Lo bicep https://github.com/Azure/bicep/releases/latest/download/bicep-linux-x64 \
+    && chmod +x ./bicep
```

Comment:
What was the reasoning behind creating a separate image for Bicep as opposed to just downloading the binary in the controller image?

Reply:
makes it easier for users to swap out the bicep container if they need to in `rad install kubernetes`. they only need to build and push the `bicep` container instead of `controller`.